komodo/check_up_to_date_pypi.py

```diff
@@ -0,0 +1,167 @@
+import copy
+import sys
+import pathlib
+
+from packaging import version as get_version
+from packaging.specifiers import SpecifierSet
+import requests
+import ruamel.yaml
+import argparse
+from komodo.package_version import LATEST_PACKAGE_ALIAS, strip_version
+from komodo.prettier import write_to_file
+
+
+def yaml_parser():
+    yaml = ruamel.yaml.YAML()
+    yaml.indent(mapping=2, sequence=4, offset=2)
+    yaml.preserve_quotes = True
+    return yaml
+
+
+def get_release(yaml, fname):
+    with open(fname, "r") as fin:
+        result = yaml.load(fin)
+    return result
+
+
+def get_packages_pypi(package_names):
+    return [
+        (package, requests.get(f"https://pypi.python.org/pypi/{package}/json"))
```

Comment:
Just out of curiosity, how long does this take? :sweat_smile:

Reply:
Shorter than you (or I at least) would think. I started shooting of more at the time, but it was not really needed I felt.

Reply:
Dont get me wrong, it is quite slow, but acceptably slow (in my opinion)